eos-tech-support/eos-diagnostics

```diff
@@ -701,7 +706,7 @@ function dumpHardwareInfo(filename) {
     });
 }
 
-function dumpCoredumps(filename) {
+function dumpCoredumps(filename, verboseFlag) {
```

Comment:
I would not have `verboseFlag` as an input parameter to this function if it is going to be ignored and hard-coded.  Actually, I think it would be better to have the function respect this input parameter and instead force passing in true in the main routine when calling this function.